plugins/aladino/builtins.go

```diff
@@ -125,6 +125,7 @@ func PluginBuiltInsWithConfig(config *PluginConfig) *aladino.BuiltIns {
 			"info":                 actions.Info(),
 			"merge":                actions.Merge(),
 			"rebase":               actions.Rebase(),
+			"review":               actions.Review(),
```

Comment:
The built-ins are in alphabetical order so the "review" built-in should be after the "removeLabels" built-in.
```suggestion
            "removeLabels":         actions.RemoveLabels(),
            "review":               actions.Review(),
```